cv_next/app/cv/[cvId]/components/cvData.tsx

```diff
@@ -11,7 +11,7 @@ export default function CvData({ cv }: { cv: CvModel }) {
     ? uploader.username
     : uploader.full_name;
   return (
-    <div className="grid grid-cols-1 gap-y-4 md:grid-cols-[65%_35%] md:gap-x-4">
+    <div className="grid grid-cols-1 gap-y-4 md:grid-cols-[70%_30%] md:gap-x-4">
```

Comment:
Confirm for me that it looks good on mobile as well?
Relevant to all changes in the PR.

Reply:
![image](https://github.com/user-attachments/assets/fa5568cd-4432-4129-b661-e627ca3c4605)